packages/module/patternfly-docs/content/extensions/component-groups/examples/ContentHeader/ContentHeaderIconExample.tsx

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-console */
+import React from 'react';
+import ContentHeader from '@patternfly/react-component-groups/dist/dynamic/ContentHeader';
+
+export const IconExample: React.FunctionComponent = () => (
+  <ContentHeader 
+    title='My Title'
+    subtitle='This is a subtitle for your content header' 
+    icon={<img src='https://console.redhat.com/apps/frontend-assets/rbac-landing/rbac-landing-icon.svg'/>}
```

Comment:
let's copy the image file to `/public` in this case rather than referencing the CRC

Reply:
also, I am not sure about the ideal icon size. It would be probably good to set up some default.
@andrew-ronaldson any thoughts? I see in HCC we use min-width of 48px
<img width="1723" alt="image" src="https://github.com/patternfly/react-component-groups/assets/50696716/15d36393-6645-440e-9729-3a6d3cfd172b">

Reply:
I like the sizing from HCC for this.

Reply:
Sounds good! thank you Andrew :-)